src/java/main/org/apache/zookeeper/server/persistence/FileTxnSnapLog.java

```diff
@@ -55,6 +53,7 @@
     private final File snapDir;
     private TxnLog txnLog;
     private SnapShot snapLog;
+    private ServerStats serverStats;
```

Comment:
`final`?

Reply:
Maybe I am missing something pretty obvious here, but it seems that `serverStats` is passed to `FileTxnLog`, but there is no place in this class where it passes `serverStats` to `FileTxnLog`.

Reply:
No, you're not missing anything, I messed up the refactor before the commit. I've written more unit tests so this will not go unnoticed next time. Will commit soon.

Thanks!